src/mimir_config.py

```diff
@@ -83,19 +89,20 @@ class Alertmanager(BaseModel):
     external_url: Optional[str]
 
 
-class _S3StorageBackend(BaseModel):
+class _S3ConfigData(BaseModel):
+    model_config = {"populate_by_name": True}
+    access_key_id: str = Field(alias="access-key")
     endpoint: str
-    access_key_id: str
-    secret_access_key: str
-    insecure: bool = False
-    signature_version: str = "v4"
+    secret_access_key: str = Field(alias="secret-key")
+    bucket_name: str = Field(alias="bucket")
+    region: str = ""
```

Comment:
why does region still have an empty string as default?

Reply:
Not a mandatory field, a user might not define a region and that shouldn't stop the connection